### DISCUSSION TITLE
Adding info of missing persons

### DISCUSSION THREAD

**Discussion by amimodr** [Joymutheu-dev/MISSING-PERSONS-DATABASE-2024-KENYA-FINANCE-BILL-PROTESTS-#7] [General]
where is the database for missing persons to add someone?
Some guru of coding can create a questionnaire like form link to the database for ease of entry

**Comment by Jojo-devv** [comment]
> where is the database for missing persons to add someone?
> Some guru of coding can create a questionnaire like form link to the database for ease of entry

Hi, here's our website lostinkenya.org
